diff --git a/node_modules/@next-auth/typeorm-legacy-adapter/dist/index.js b/node_modules/@next-auth/typeorm-legacy-adapter/dist/index.js
index 733ee42..a1ebb9c 100644
--- a/node_modules/@next-auth/typeorm-legacy-adapter/dist/index.js
+++ b/node_modules/@next-auth/typeorm-legacy-adapter/dist/index.js
@@ -101,7 +101,7 @@ function TypeORMLegacyAdapter(configOrString, options = {}) {
 
       if (config.type === "mongodb") {
         idKey = "_id";
-        const mongodb = (0, _require_optional.default)("mongodb");
+        const mongodb = require('mongodb');
         ObjectId = mongodb.ObjectId;
       }
 
